ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,297 @@
+#!/bin/bash
+char="#"
+debug=''
+sys_type="yum"
+
+logger() {
+
+    echo $1
+}
+
+startService() {
+
+    if [ -n "$(ps -e | egrep ^\ *1\ .*systemd$)" ]; then
+        systemctl daemon-reload 
+        systemctl enable $1.service 
+        systemctl start $1.service 
+        if [ "$?" != 0 ]
```

Comment:
Unify the if notation in the script, either 
```
if [ ]
then
```
or 

```
if [ ]; then
```

Reply:
Done in 039d5eee2611844622e77f70f8d01bfbf9221dcb